sql/core/src/test/resources/sql-tests/inputs/typeCoercion/native/decimalArithmeticOperations.sql

```diff
@@ -49,7 +49,7 @@ select 1e35 / 0.1;
 
 -- arithmetic operations causing a precision loss are truncated
 select 123456789123456789.1234567890 * 1.123456789123456789;
-select 0.001 / 9876543210987654321098765432109876543.2
+select 0.001 / 9876543210987654321098765432109876543.2;
```

Comment:
... A good catch! We need to review the PR more carefully.

Reply:
This is not a good test case. The results and schemas are the same no matter whether we set  `spark.sql.decimalOperations.allowPrecisionLoss` to true or false.